src/ducks/userInfo/userInfoActions.jsx

```diff
@@ -79,4 +87,17 @@ export const fetchGoogleData = () => {
                 alert(error.message);
             });
     }
+}
+
+export const fetchLogout = () => {
+    return dispatch => {
+        axios
+            .post(`https://kidslike-v1-backend.goit.global/auth/logout`)
```

Comment:
I think you need to pass token into header, otherwise server wont be able to logout you

Reply:
You should pass token as Authorization header, not like body

Reply:
It seems to me that this is how you can specify the request header. However, I still look forward to your review?))
![image](https://user-images.githubusercontent.com/96341034/189667314-16e4743c-c7e3-4490-a699-494026dab246.png)

Reply:
where do you get user info here? Its not defined

Reply:
Can I create a constant and use it?

export const fetchLogout = () => {
    const userAuthorized = store.getState().userInfo.token
    return dispatch => {
        axios
            .post(`https://kidslike-v1-backend.goit.global/auth/logout`, {
                headers: {
                    'Authorization': userAuthorized
                }
            })
=================================================
Or can specify a value in - payload?